website/docs/data-display/chart-legend/examples/table-view.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ChartLegend, LegendTableProps } from '@semcore/d3-chart';
+import { Text } from '@semcore/typography';
+
+const data = [...Array(5).keys()].map((d, i) => ({
+  x: i,
+  Line1: Math.random() * 10,
+  Line2: Math.random() * 10,
+  Line3: Math.random() * 10,
+  Line4: Math.random() * 10,
+  Line5: Math.random() * 10,
+}));
+
+export default () => {
+  const lines = Object.keys(data[0])
+    .filter((key) => key !== 'x')
+    .reduce<LegendTableProps['items']>((res, item, index) => {
```

Comment:
Why do you do `.map` with `.reduce`? 😁

Reply:
copy-paste from some first place, where i had a `reduce`. sorry)